### DISCUSSION TITLE
How can we log in ambianic-edge.img installed on Raspberry?

### DISCUSSION THREAD

**Discussion by BBIGIT** [ambianic/ambianic-ui#828] [Q&A] [answered]
How can we log in ambianic-edge.img installed on Raspberry? 
![IMG_1100](https://user-images.githubusercontent.com/90552134/161947861-d6865358-a31e-466f-a233-89811a00d9ae.JPG)

**Comment by ivelin** [comment, marked as answer]
Can you please elaborate? What do you mean by logging an image?

**Reply by BBIGIT** [reply]
I took this photo when I turn on the Raspberry.  The Raspberry told me that I need to log in. I need to type the id and password. How can I get this account?

**Reply by BBIGIT** [reply]
![8650DC71A24D991C5913C87CBCF9B861](https://user-images.githubusercontent.com/90552134/162003418-73424157-050a-45ec-a8aa-0c6d3205c05e.png)

**Reply by ivelin** [reply]
Thank you for clarifying. Take a look at the docs. Default access creds are in the `Developing and Debugging` section.
https://docs.ambianic.ai/users/ambianicos/#first-boot-network-setup

**Reply by BBIGIT** [reply]
We've succeeded!  THX for your sincere help! Have a nice day !!!

**Reply by BBIGIT** [reply]
Hi! We met another problem. 
We've studied the code from `https://github.com/ambianic/ambianic-edge` and got Ambianic Edge binaries and data files under `/opt/ambianic` and config files under `/etc/ambianic` installed on Raspberry. 
So the problem is that we want to find the contact between this two codes. We want to know how can we understand the code running on the Raspberry.

**Reply by BBIGIT** [reply]
Dear author! May I get the source code of the `https://ui.ambianic.ai` ?
I would appreciate it if you can help us !!!!
